### DISCUSSION TITLE
Dịch `Voting classifier`, `Hard voting classifier`

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#43] [Terminology] [answered]
Hai từ này xuất hiện trong chương 7 và liên quan đế các thuật toán dựa trên Ensemble. Hiện tại mình đang dịch như sau:

- `Voting classifier`: `Bộ phân loại Biểu quyết`
- `Hard voting classifier`: `Bộ phân loại biểu quyết cứng`

Mọi người cho ý kiến nhé @mlbvn/handson-ml2.
Context: 

![image](https://user-images.githubusercontent.com/41325216/109947420-95542800-7cd9-11eb-8dc3-83074fd846c0.png)

**Comment by thanhcsf** [comment, marked as answer]
Chốt dịch như ông giáo @quangnhat185 đề xuất anh em nha.
- `Voting classifier`: `Bộ phân loại Biểu quyết`
- `Hard voting classifier`: `Bộ phân loại biểu quyết cứng`
